code/__DEFINES/flags.dm

```diff
@@ -72,6 +72,7 @@
 #define EASYDISMEMBER	20
 #define EASYLIMBATTACHMENT 21
 #define TOXINLOVER		22
+#define DIGITIGRADE		23	//Uses weird leg sprites. Optional for Lizards, required for ashwalkers. Don't give it to other races unless you make sprites for this (see human_parts_greyscale.dmi)

```

Comment:
@Bawhoppen all furries are sex-crazed perverts

they seek to forcefully insert their fetishes into everything they come into contact with

@optimumtact this isnt something you're going to know off the top of your head, nor is it going to be obvious

"keep it simple, stupid" is a great practice for backend shit

i dont mind this being the frontend name, but in the back end it should be simple